packages/graphql-transformer-core/src/util/transformConfig.ts

```diff
@@ -28,7 +28,18 @@ export interface TransformConfig {
      */
     StackMapping?: {
         [resourceId: string]: string
-    }
+    },
+
+    /**
+     * Provide build time options to GraphQL Transformer constructor functions.
+     * Certain options cannot be configured via CloudFormation parameters and
+     * need to be set at build time. E.G. DeletionPolicies cannot depend on parameters.
+     */
+    TransformerOptions?: {
```

Comment:
What are the list of options that could be passed out here?

Reply:
The intention is to have each transformer define its own set of options that can be passed. Right now the only acceptable option is like:

```
{
  "@model": {
    "EnableDeletionProtection": true/false
  }
}
```